lib/champaign_queue.rb

```diff
@@ -1,19 +1,23 @@
 # frozen_string_literal: true
 require_relative 'champaign_queue/clients/sqs'
-require_relative 'champaign_queue/clients/direct'
+require_relative 'champaign_queue/clients/http'
 
 module ChampaignQueue
   extend self
 
   def push(opts)
-    if Rails.env.production?
+    if Rails.env.production? || Settings.publish_champaign_events
       client.push(opts)
     else
       false
     end
   end
 
   def client
-    Clients::Sqs
+    if Settings.champaign_queue_client == 'http'
```

Comment:
In the previous version, the kind of client was not configurable, it was always Sqs. In the new version you can set the client to be the HTTP direct client via a config setting.